http://gcc.gnu.org/bugzilla/show_bug.cgi?id=14944

2004-04-28  Serge Belyshev  <1319@bot.ru>

	PR 14944
	* coverage.c (read_counts_file): Fix usage of warning () call.
	* pretty-print.c (pp_base_format_text): Fix typo in the comment.

===================================================================
RCS file: /cvs/gcc/gcc/gcc/coverage.c,v
retrieving revision 1.33
retrieving revision 1.34
diff -u -r1.33 -r1.34
--- gcc/gcc/coverage.c	2004/03/23 23:47:35	1.33
+++ gcc/gcc/coverage.c	2004/04/29 06:28:59	1.34
@@ -172,8 +172,8 @@
       GCOV_UNSIGNED2STRING (v, tag);
       GCOV_UNSIGNED2STRING (e, GCOV_VERSION);
 
-      warning ("`%s' is version `%.4s', expected version `%.4s'",
- 	       da_file_name, v, e);
+      warning ("`%s' is version `%.*s', expected version `%.*s'",
+ 	       da_file_name, 4, v, 4, e);
       gcov_close ();
       return;
     }
===================================================================
RCS file: /cvs/gcc/gcc/gcc/pretty-print.c,v
retrieving revision 2.10
retrieving revision 2.11
diff -u -r2.10 -r2.11
--- gcc/gcc/pretty-print.c	2004/03/24 23:45:41	2.10
+++ gcc/gcc/pretty-print.c	2004/04/29 06:28:59	2.11
@@ -178,7 +178,7 @@
    %p: pointer.
    %m: strerror(text->err_no) - does not consume a value from args_ptr.
    %%: `%'.
-   %*.s: a substring the length of which is specified by an integer.
+   %.*s: a substring the length of which is specified by an integer.
    %H: location_t.  */
 void
 pp_base_format_text (pretty_printer *pp, text_info *text)
